mod.ts

```diff
@@ -11,6 +11,7 @@ import {
 } from "./deps.ts";
 import { CSS, KATEX_CLASSES, KATEX_CSS } from "./style.js";
 export { CSS, KATEX_CSS, Marked };
+import "https://esm.sh/prismjs@1.29.0/components/prism-yaml";
```

Comment:
Is pulling in yaml for everyone necessary? It seems like the actual fix is the additional types below. Won't that still work if the user app pulls in YAML (like it has to pull in most languages)?

Reply:
Sadly I think it is. The necessary thing is making sure this returns something:
https://github.com/denoland/deno-gfm/blob/b9f7352e618138d62201a245a9f6a44e23a2ba6d/mod.ts#L53

The only way I could do that is by importing the package. For whatever reason I couldn't find any way to invoke the loadlanguages function that prism has.